### PR TITLE
Upgrade to ROS2 (Galactic)

### PR DESCRIPTION
**Pull request by wuehr1999**
Hi @aentinger,
I managed to get a joystick device and tested the ROS2-node successfully. How do we proceed regarding the maintenance of both versions for ROS1 and ROS2?

### CODE REVIEW

**Comment by aentinger** [comment]
My bad, I did see the branch but I was never notified of the PR. Will get to check on this asap.

File: launch/joy.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description():
```
Comment (aentinger) [inline launch/joy.py:4]: Just being curious, what's the advantage of generating the launch file via Python vs the "old" way?

**Comment by aentinger** [comment]
>I managed to get a joystick device and tested the ROS2-node successfully.

Great work! :bow: 

>How do we proceed regarding the maintenance of both versions for ROS1 and ROS2?

`main` will become the branch for ROS2 development, there's also the branch [ros-noetic-ninjemys](https://github.com/107-systems/l3xz_teleop/tree/ros-noetic-ninjemys) which can be used for further ROS1 development.

File: launch/joy.py (hunk 1 above)
Comment (wuehr1999) [reply launch/joy.py:4]: The big advantage is, that it is possible to use all Python features in the launch file. Anyway, the syntax of the launch file would make no difference for this application ;-)

**Comment by wuehr1999** [comment]
Hi @aentinger 
I just checked the funciontality of the teleop node and everything is fine

File: launch/joy.py (hunk 1 above)
Comment (aentinger) [reply launch/joy.py:4]: Good to know, I'll have to look into it more =) Thank you :bow:

**Pull request merged**